docs/reference/filebeat/filebeat-input-filestream.md

```diff
@@ -560,6 +560,26 @@ The `clean_inactive` configuration option is useful to reduce the size of the re
 
 This config option is also useful to prevent Filebeat problems resulting from inode reuse on Linux. For more information, see [Inode reuse causes Filebeat to skip lines](/reference/filebeat/inode-reuse-issue.md).
 
+To disable, set `clean_inactive: -1`. {applies_to}`stack: ga 9.2.0`Setting it to zero will also
+disable it.
+
+::::{warning}
+For versions `<= 9.2.0` setting `clean_inactive: 0` will cause
+Filebeat to re-ingest all files on restart.
+::::
+
+{applies_to}`stack: ga 9.2.0` Filebeat enforces the restrictions by
+failing to start if `clean_inactive <= ignore_older +
+prospector.scanner.check_interval` or if `ignore_older` is disabled.
+
+{applies_to}`stack: ga 9.2.0` To restore the old behaviour of not enforcing the
+configuration restricition and re-ingesting files if `clean_inactive:
+0`, set `legacy_clean_inactive: true`.
```

Comment:
This is kind of a cheat, but what if we just combined these two sentences into a single paragraph? This is not a solution that's going to hold up in all future situations, but I don't think it's a stretch to combine these in this particular case. 

<img width="720" height="121" alt="Screenshot 2025-09-05 at 5 36 42 PM" src="https://github.com/user-attachments/assets/553247a6-177b-4dc6-a487-28c62eced30a" />

```suggestion
{applies_to}`stack: ga 9.2.0` Filebeat enforces the restrictions by
failing to start if `clean_inactive <= ignore_older +
prospector.scanner.check_interval` or if `ignore_older` is disabled.
To restore the old behaviour of not enforcing the
configuration restriction and re-ingesting files if `clean_inactive:
0`, set `legacy_clean_inactive: true`.
```